account_banking_payment_export/wizard/payment_order_create.py

```diff
@@ -45,9 +46,33 @@ def default_get(self, field_list):
     def extend_payment_order_domain(self, payment_order, domain):
         if payment_order.payment_order_type == 'payment':
             domain += [('account_id.type', 'in', ('payable', 'receivable')),
-                       ('amount_to_pay', '>', 0)]
+                       ('credit', '>', 0)]
```

Comment:
OK, I'm still not sure about this serving for refunds, but I need to investigate. In direct debit we will need something similar (remove amount_to_receive). Doesn't it?